TUTORIAL.md

```diff
@@ -570,6 +570,24 @@ We refer to whatever comes before a `->` in a `when` expression as a *pattern* -
 patterns in branching conditionals like `when` is known as [pattern matching](https://en.wikipedia.org/wiki/Pattern_matching). You may hear people say things like "let's pattern match on `Custom` here" as a way to
 suggest making a `when` branch that begins with something like `Custom description ->`.
 
+### Pattern Matching on Lists
+You can also pattern match on lists, like so:
+
+    when myList is
+        [] -> # the list is empty
+        [Foo, ..] -> # it starts with a Foo tag
+        [Foo, Bar, ..] -> # it starts with a Foo tag followed by a Bar tag
+        [Foo, a, ..] -> # it starts with a Foo tag followed by something we name `a`
+        [Ok a, ..] -> # it starts with an Ok tag containing a payload named `a`
+        [.., Foo] -> # it ends with a Foo tag
+        [A, B, .., C, D] -> # it has certain elements at the beginning and and
```

Comment:
maybe worth adding a note that this is not valid Roc code, just illustrative?

Reply:
Consider also including the following patterns:

```
[_, ..] -> # the list starts with an element, which we ignore
[A, B, C] -> # the list contains exactly three elements, A, B, C
```

Reply:
Cool! Updated in 99b48c31628d123392e93ccfabb8fa4ce7a86b1e - how's that look?